netlogo-gui/src/main/app/App.scala

```diff
@@ -271,11 +270,7 @@ class App extends
   private var _workspace: GUIWorkspace = null
   def workspace = _workspace
   lazy val owner = new SimpleJobOwner("App", workspace.world.mainRNG, AgentKind.Observer)
-  private var _tabs: Tabs = null
-  private var _codeTabsPanel: CodeTabsPanel = null
-  private var _tabManager : AppTabManager= null
-  def tabs = _tabs
-  def codeTabsPanel = _codeTabsPanel
+  private var _tabManager: TabManager = null
```

Comment:
The underscore allows the getter function below, `def tabManager = _tabManager`, which definitely shouldn't be changed or it will have to be changed in tons of other places in the code, including the ls extension. What would you recommend the private variable be named instead?

Reply:
You're right.  What you did is the standard naming convention for what you've described.  Complaint retracted.